### PR TITLE
Backlog updates fix

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
This PR includes the following changes:

- Addition of the `zendesk__ticket_backlog` model and accompanying documentation. This model includes the fields and metrics outlines within the [Zendesk Backlog History](https://support.zendesk.com/hc/en-us/articles/360022182114#topic_et2_myf_kkb) dataset.

## Joe's PR Overview Loom Video
[image](https://www.loom.com/share/f8940c6774514e5f8e924f86a7b4f9b8)

### CODE REVIEW

**Comment by fivetran-joemarkiewicz** [comment]
I noticed CircleCi failed after the PR was initiated. Not entirely sure what the issue is, but will continue to look into it.

File: models/zendesk__ticket_backlog.sql [hunk 1]
```diff
@@ -0,0 +1,99 @@
+with ticket_field_history as (
```
Comment (fivetran-kristin) [inline models/zendesk__ticket_backlog.sql:1]: enable this model only if status is in field_history list

File: models/zendesk__ticket_backlog.sql [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+), organizations as (
+    select *
+    from {{ ref('stg_zendesk__organization') }}
+
+), backlog as (
+    select
+        ticket_field_history.date_day
+        ,tickets.created_channel
+        {% set field_count = var('ticket_field_history_columns')|length + 2 %} --Adding 2 in order to include date_day and created_channel in group by
```
Comment (fivetran-kristin) [inline models/zendesk__ticket_backlog.sql:36]: remove this and don't aggregate

Comment (fivetran-joemarkiewicz) [reply models/zendesk__ticket_backlog.sql]: Updated

File: models/zendesk__ticket_backlog.sql (hunk 1 above)
Comment (fivetran-joemarkiewicz) [reply models/zendesk__ticket_backlog.sql:4]: Updated.

**Review by fivetran-kristin — APPROVED** [review]
Once Circle has been fixed, feel free to merge 👍

**Pull request merged**